templates/partial/_get-started-windows.html

```diff
@@ -1,8 +1,8 @@
-<div class="row--25-75">
-  <div class="col">
-    <img src="https://assets.ubuntu.com/v1/11ecbd2d-2018-logo-windows.svg" style="height: 145px;" alt="Windows">
+<div class="row">
+  <div class="col-3 u-hide--medium u-hide--small p-image-wrapper">
+    <img src="https://assets.ubuntu.com/v1/2c5b3fa9-windows-logo.svg" style="height: 48px;" alt="Windows">
```

Comment:
Image template. For styling attribute you could do:

```

{{ image (
  url="https://assets.ubuntu.com/v1/2c5b3fa9-windows-logo.svg",
  alt="Windows",
  width="316",
  height="72",
  hi_def=True,
  loading="auto",
  attrs={"style": "height: 48px;"}
  ) | safe
}}
```

Or of course you could use a classname and add styles to a stylesheet the same way.